### PR TITLE
OSDOCS-610: Updated details regarding raw block volumes.

### PR DESCRIPTION
**Pull request by huffmanca**
Updated existing information on raw block volumes, and provided details on full support for 4.2 raw block volumes.

This is for OS 4.2.

### CODE REVIEW

**Comment by huffmanca** [comment]
@openshift/team-documentation , 

Is someone able to review this one?

File: modules/storage-persistent-storage-block-volume.adoc [hunk 1]
```diff
@@ -8,8 +8,13 @@
 [id="block-volume-support_{context}"]
 = Block volume support
 
-You can statically provision raw block volumes by including API fields
-in your PV and PVC specifications.
+{product-title} can statically provision raw block volumes. These volumes
+do not have a filesystem, and can provide performance benefits for
```
Comment (bergerhoffer) [inline modules/storage-persistent-storage-block-volume.adoc]: filesystem -> file system

**Review by bergerhoffer — COMMENTED** [review]
One minor thing, otherwise LGTM!

File: modules/storage-persistent-storage-block-volume-examples.adoc [hunk 2]
```diff
@@ -67,12 +69,13 @@ spec:
       persistentVolumeClaim:
         claimName: block-pvc <3>
 ----
-<1> `volumeDevices`, similar to `volumeMounts`, is used for block devices
-and can only be used with `PersistentVolumeClaim` sources.
-<2> `devicePath`, similar to `mountPath`, represents the path to the
-physical device.
+<1> `volumeDevices`, instead of `volumeMounts`, is used for block
```
Comment (vikram-redhat) [inline modules/storage-persistent-storage-block-volume-examples.adoc]: @huffmanca is the build failing because the callouts are before the text?

Comment (huffmanca) [reply modules/storage-persistent-storage-block-volume-examples.adoc]: It was failing due to an issue in the block volume table, where I was attempting to include the callout stanza after the table without defining them inside it.

This issue should be resolved.

**Comment by vikram-redhat** [comment]
@liangxia PTAL

**Comment by liangxia** [comment]
LGTM (from QE)

**Pull request merged**